XMLProjectIS/Pages/Index.cshtml.cs

```diff
@@ -24,8 +24,21 @@ public void OnGet()
             using (var webClient = new WebClient())
             {
                 string jsonString = webClient.DownloadString("https://raw.githubusercontent.com/prust/wikipedia-movie-data/master/movies.json");
-                var welcome = Welcome.FromJson(jsonString);
-                ViewData["Welcome"] = welcome;
+                string result = "";
+
+                try
+                {
+                    result = webClient.DownloadString(jsonString);
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine("Exception while calling API", e);
+
+
+
+                }
+                var movieList = Movies.MovieCollection.FromJson(jsonString);
```

Comment:
Logging is a good idea, but what should you do with the exception at this point?  Re-throw it?  If you just log it and bury it, the program will continue to execute after the exception occurs, as if nothing went wrong.  That is worse than the default behavior: terminate the method when an exception occurs.

Use proper logging instead of Console.WriteLine().
- Logging has the concept of priority (error, debug, warn, etc.), which Console.WriteLine() doesn't have.
- No one will see Console.WriteLine(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine().